poetry/core/masonry/utils/include.py

```diff
@@ -46,3 +46,24 @@ def refresh(self):  # type: () -> Include
         self._elements = sorted(list(self._base.glob(self._include)))
 
         return self
+
+
+class IncludeFile:
```

Comment:
This hould probably be in builder module, and could be renamed to something like `BuildIncludeFile` to be more descriptive of purpose.